dev/archery/archery/benchmark/compare.py

```diff
@@ -25,11 +25,11 @@ def items_per_seconds_fmt(value):
     if value < 1000:
         return "{} items/sec".format(value)
     if value < 1000**2:
-        return "{:.3f}k items/sec".format(value / 1000)
+        return "{:.3f}K items/sec".format(value / 1000)
```

Comment:
was this an intended change in this PR? I don't know anything about the archery benchmark driver, so I think someone who does should probably review these changes if they are needed in this PR

Reply:
probably caused by force push from master, i will rebase.